acceptance_tests/tests/base.py

```diff
@@ -367,3 +368,22 @@ def assert_same_dir_content(self, a, b):
             self.assertTrue(False)
         logfile.write("OK\n")
         logfile.close()
+
+    def assert_file_contains_line(self, path_to_file, line):
+        contents = ""
+        found = False
+
+        if path_to_file.endswith(".gz"):
+            with gzip.open(path_to_file, 'rb') as f:
+                for l in f.readlines():
+                    if (l.strip() == line.strip()):
+                        found = True
+        self.assertTrue(found, "Line: " + line + " not found in file: " + path_to_file)
+        return
+
+        with open(path_to_file) as f:
```

Comment:
Code after return? Dead code?

Reply:
Oh, my mistake, I have updated it.
Thank you.